task/jefe/src/main.rs

```diff
@@ -295,8 +294,13 @@ impl idol_runtime::NotificationHandler for ServerImpl<'_> {
         if bits & notifications::TIMER_MASK != 0 {
             // If our timer went off, we need to reestablish it
             if sys_get_timer().now >= self.deadline {
-                self.deadline += TIMER_INTERVAL;
-                sys_set_timer(Some(self.deadline), notifications::TIMER_MASK);
+                // Wrapping addition because our deadline is derived from the
+                // uptime, which will not approach wrapping a u64 for another
+                // 584 million years.
```

Comment:
there isn't a wrapping add here --- is this comment intended to refer to what `set_timer_relative` actually does?